lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -52,13 +52,13 @@ public static function assertValidIdentifier($identifier)
     /**
      * {@inheritDoc}
      */
-    public function getSubstringExpression($value, $position, $length = null)
+    public function getSubstringExpression($value, $from, $length = null)
```

Comment:
Please make sure this is consistent with `master` (https://github.com/doctrine/dbal/pull/3494). The idea is to have the arguments match ANSI SQL analogs if possible (e.g.[ `SUBSTR()`](https://www.w3schools.com/sql/func_mysql_substr.asp)).